package.json

```diff
@@ -363,6 +363,9 @@
     "@storybook/react-webpack5": "7.6.9",
     "@storybook/source-loader": "7.0.20",
     "@storybook/theming": "7.0.20",
+    "@swc-node/register": "~1.9.1",
+    "@swc/core": "~1.5.7",
+    "@swc/helpers": "~0.5.11",
```

Comment:
This seems to be included when generating new projects after the nx upgrade. Could you pin the versions please?